### PR TITLE
Weather app - [COMPLETED]

### PR DESCRIPTION
**Pull request by marcos-medeiros**
Thanks for taking the time to review my code.

Done work:

- [x] Create input to get the city name.

- [x] Create div to display the city's data.

- [x] Create button to convert temperature.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,4 +1,49 @@
...
+  if (changeTempBtn.innerHTML.includes('Fahrenheit')) {
+    tempHolder = (tempHolder * (9 / 5)) + 32;
+    temp.innerHTML = `Temperature: ${tempHolder.toFixed(2)} Fahrenheit`;
+    changeTempBtn.innerHTML = 'Change temperature to Celsius';
+  } else {
+    tempHolder = (tempHolder - 32) * (5 / 9);
+    temp.innerHTML = `Temperature: ${tempHolder.toFixed(2)} Celsius`;
+    changeTempBtn.innerHTML = 'Change temperature to Fahrenheit';
+  }
+};
```
Comment (Oghenebrume50) [inline src/index.js]: Please make a request for both the celsius data and Fahrenheit data see this

File: src/index.js [hunk 2]
```diff
@@ -1,4 +1,49 @@
...
+const description = document.getElementById('description');
+const temp = document.getElementById('temp');
+const humidity = document.getElementById('humidity');
+const changeTempBtn = document.getElementById('change-temp');
+const weatherDiv = document.getElementById('weather-data');
+let tempHolder = 0;
+
+const getWeatherData = async (cityValue) => {
+  const response = await fetch(`https://api.openweathermap.org/data/2.5/forecast?id=524901&APPID=52119e88a31d7d889fe0c2dc770ee44b&units=metric&q=${cityValue}`);
+  const weatherJSON = await response.json();
```
Comment (Oghenebrume50) [inline src/index.js]: There is  no error handling for the async-await, please add error hadlndling see this

File: src/index.js [hunk 3]
```diff
@@ -1,4 +1,49 @@
...
+  await displayWeatherData(weatherJSON);
+};
+
+const displayWeatherData = (data) => {
```
Comment (Oghenebrume50) [inline src/index.js]: You can use object destructuring here to retrieve values from the data object see this

File: README.md [hunk 4]
```diff
@@ -1 +1,46 @@
...
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+Open your git bash and cd to the location you'd like to put your files the run the command below.
+
+```console
+git clone https://github.com/marcos-medeiros/weather-app.git
```
Comment (bellom) [inline README.md]: update your live link !!

**Pull request merged**